genisys/templates/preseed.cfg.jinja2

```diff
@@ -94,3 +94,10 @@ in-target chmod 644 /root/.ssh/authorized_keys; \
 in-target chmod 700 /root/.ssh/
 {% endif %}
 
+# copy firstboot.service file to client 
+d-i preseed/late_command string cp -r /ftp/first-boot/genisys-firstboot.service /etc/systemd/system/
```

Comment:
Remember that this command is run on the client, but the FTP directory is on the server. This command here will fail because the /ftp directory doesn't exist on the client.

You should be using the `ftp` command along with the ip and port specified in the config to download these files from the genisys server.